doc/Development_Documentation/19_Development_Tools_and_Details/09_Cache/03_Full_Page_Cache.md

```diff
@@ -41,7 +41,7 @@ pimcore:
 | ------ | ----------- |
 | Enable | Set to true to enable to full page cache. |
 | Lifetime | You can optionally define a lifetime (in seconds) for the  full page cache. If you don't do, the cache is evicted automatically when there is a modification in the Pimcore Backend UI. If there is a lifetime the item stays in the cache even when it is changed until the TTL is over. The lifetime is useful if you have embedded some items which are not directly in the cms, like rss feeds, or twitter messages over the API. It is also highly recommended to specify a lifetime on high traffic websites so that the frontend (caches) isn't affected by changes in the admin-UI. Otherwise on every change in the admin-UI the whole output-cache is flushed, what can have drastic effects to the server environment. |
-| Exclude Patterns | You can define some exclude patterns where the cache doesn't affect. The patterns have to be valid regular expressions (including delimiters). Type one pattern in each line. |
+| Exclude Patterns | You can define some exclude patterns where the cache doesn't affect. The patterns have to be valid regular expressions (including delimiters). seperate patterns with , |
```

Comment:
```suggestion
| Exclude Patterns | You can define some exclude patterns where the cache doesn't affect. The patterns have to be valid regular expressions (including delimiters). seperate patterns with `,` |
```

Reply:
The word after the dot (end of phrase) looks grammarly incorrect, what about
```suggestion
| Exclude Patterns | You can define some exclude patterns where the cache doesn't affect. The patterns have to be valid regular expressions (including delimiters) and different patterns should be seperated by `,` |
```

Reply:
But i have an input: should be changing the config node to work as arrayNode instead and BC break in 11?
What if the regex pattern has a range like `{1,4}`, the implode could break by this comma, isn't it?